pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3607,6 +3607,9 @@
         "4.5.0",
         "4.5.1"
       ],
+      "InboxOn": {
+        "uap10.0.16300": "4.0.4.0"
```

Comment:
This was removed as part of https://github.com/dotnet/corefx/pull/29970 so I don't think it should be added back.